rootfs_linux.go

```diff
@@ -62,6 +69,21 @@ func setupRootfs(config *configs.Config, console *linuxConsole) (err error) {
 	return nil
 }
 
+func mountCmd(cmd string) error {
+	if cmd != "" {
+		args := strings.Split(cmd, " ")
+		mCmd := exec.Command(args[0], args[1:]...)
```

Comment:
If you want to capture output in case of failure and return it back:
```go
output, err := exec.Command(args[0], args[1:]...).CombinedOutput()
if err != nil {
       return fmt.Errorf("Mounting failed: %v, err, output)
}
```
If not, then Run is fine like you have it now.